packages/website/docs/plugins.md

```diff
@@ -3,21 +3,244 @@ id: plugins
 title: Plugins
 ---
 
-:::note Draft
+Plugins encapsulate and distribute custom Autocomplete behaviors.
 
-This page needs to cover:
+An autocomplete can be much more than a functional combo box. **Autocomplete lets you extend and encapsulate custom behavior with its Plugin API.**
 
-- Plugins are a way of encapsulating sources, including the data and display. They can also trigger particular actions on an interaction.
-- For example, the insights plugin sends click and conversion events.
-- We provide some out-of-the-box plugins, but you can also build your own.
-- As a simple example, you could create a plugin to display a static list of predefined items:
-  - Code snippet
-- We also have created a couple out-of-the box plugins:
-  - Our **recent-searches plug-in** stores recently made searches in local storage so that you can incorporate an individual’s recent searches into an autocomplete.
-    - Code snippet + live example
-  - Our **query-suggestions plug-in** is made to work with an Algolia Query Suggestions index.
-    - Code snippet + live example
+For example, the [official Algolia Insights plugin](createAlgoliaInsightsPlugin) automatically sends click and conversion events to the [Algolia Insights API](https://www.algolia.com/doc/rest-api/insights/) whenever a user interacts with the autocomplete.
 
-We'll also need to document the Plugin API.
+You can use one of our existing official plugins or build your own.
```

Comment:
I don't know if we're using a different tone in these docs, but considering we'll probably one day move them algolia.com/docs/ I've been trying to stay away from Vale issues as much as I'm mentally capable of. :) 

```suggestion
You can use one of the existing official plugins or build your own.
```

Reply:
You're right!